tests/output/ConsoleInteractionTest.py

```diff
@@ -151,23 +151,27 @@ def tearDown(self):
     def test_require_settings(self):
         self.assertRaises(TypeError, acquire_settings, self.log_printer, 0)
 
+        curr_section = Section("")
```

Comment:
can we have a test that verifies that the "from section Default" is present in the output string?